target/linux/sunxi/patches-5.4/450-arm64-dts-enable-wifi-on-pine64-boards.patch

```diff
@@ -0,0 +1,72 @@
+--- linux-5.4.99/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts.orig	2021-02-17 10:35:20.000000000 +0100
++++ linux-5.4.99/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine-baseboard.dts	2021-02-28 12:41:51.658785725 +0100
```

Comment:
Please refresh this patch with ```make target/linux/{clean,refresh} V=99```, otherwise it looks good.

Reply:
Thanks - refreshed and pushed to master.